lnwallet/interface.go

```diff
@@ -428,7 +428,7 @@ type WalletController interface {
 	// does not perform any other tasks (such as coin selection, UTXO
 	// locking or input/output/fee value validation, PSBT finalization). Any
 	// input that is incomplete will be skipped.
-	SignPsbt(packet *psbt.Packet) error
+	SignPsbt(packet *psbt.Packet) ([]int32, error)
```

Comment:
Any reason to not make this `[]uint32` instead? Since we only add the indices of the inputs we signed, we won't have the need for a `-1` value as we sometimes do in other places. And I'm pretty sure protobuf also supports `repeated uint32` types.